gammapy/cube/make.py

```diff
@@ -257,3 +297,359 @@ def _check_selection(selection):
             raise ValueError("Selection not available: {!r}".format(name))
 
     return selection
+
+
+class MapMakerRing(MapMaker):
+    """Make maps from IACT observations.
+
+    The main motivation for this class in addition to the `MapMaker`
+    is to have the common image background estimation methods,
+    like `~gammapy.background.RingBackgroundEstimator`,
+    that work using on and off maps.
+
+    To ensure adequate statistics, only observations that are fully
+    contained within the reference geometry will be analysed
+
+    Parameters
+    ----------
+    geom : `~gammapy.maps.WcsGeom`
+        Reference image geometry
+    offset_max : `~astropy.coordinates.Angle`
+        Maximum offset angle
+    exclusion_mask : `~gammapy.maps.Map`
+        Exclusion mask
+    background_estimator : `~gammapy.background.RingBackgroundEstimator`
+        or `~gammapy.background.AdaptiveRingBackgroundEstimator`
+        Ring background estimator or something with an equivalent API.
+
+    """
+
+    def __init__(
+        self, geom, offset_max, exclusion_mask=None, background_estimator=None
+    ):
+        super(MapMakerRing, self).__init__(
```

Comment:
In Python 3 `super` can be used without passing `self`. Just write `super().__init__`